streampark-common/src/test/scala/org/apache/streampark/common/util/YarnTest.scala

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.common.util
+
+import org.apache.hadoop.yarn.api.records.{NodeReport, NodeState}
+import org.apache.hadoop.yarn.exceptions.YarnException
+import org.junit.jupiter.api.Test
+
+import java.io.{File, IOException}
+
+class YarnTest {
+
+  @Test
+  @throws[IOException]
+  @throws[YarnException]
+  def vcore(): Unit = {
+    val numYarnMaxVcores = HadoopUtils.yarnClient.getNodeReports(NodeState.RUNNING).stream.mapToInt(report => report.getCapability.getVirtualCores).max.orElse(0)
+    System.out.println(numYarnMaxVcores)
+  }
+
+  @Test def getURL(): Unit = {
+    /**
+     * 将hadoop的配置文件放到一个目录下,
+     * 在运行该类的时候加上jvm级别的参数(idea里的 vmOption ) -DHADOOP_CONF_DIR=${目录}
+     */
+    val url = YarnUtils.getRMWebAppURL
```

Comment:
@RocMarshal Thanks for you feedback. Could you handle these non-ascaii characters by remove or translate ways?